### PR TITLE
Add note about `canvaskit-wasm` to README

### LINKED ISSUE #19: Astro requires `canvaskit-wasm` as a direct dependency
While implementing this package in an Astro static build, I struggled to make `astro-og-canvas` work, until I found, randomly this in the contributor's guide for Astro's Docs site which uses this same package, but works.

https://github.com/withastro/docs/blob/7f93e916436aee2724b99f63436d826f1f345f7e/contributor-guides/astro-maintainers-guide-to-docs.md?plain=1#L164

I guess removing this dependency might be complicated, but what do you all think of adding a note to the readme file?

### PR DESCRIPTION
**Pull request by itsmatteomanf**
Closes https://github.com/delucis/astro-og-canvas/issues/19.

### CODE REVIEW

File: packages/astro-og-canvas/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This package provides utlities to generate OpenGraph images for the pages on your Astro site.
 
-## Installation
+## Installation[^1]
```
Comment (delucis) [inline packages/astro-og-canvas/README.md]: I think the footnote format may not be the clearest as it forces a reader to jump around the page.

I’d probably add a comment immediately below the install instructions, so we have something like:

````md
## Installation

```shell
npm i astro-og-canvas
```

**Using `pnpm`?** `pnpm` users will also need to install `canvaskit-wasm` as a direct dependency:

```shell
pnpm i canvaskit-wasm
```
````

Comment (itsmatteomanf) [reply packages/astro-og-canvas/README.md]: Done!

**Review by delucis — APPROVED** [review]
Looks great, thanks @itsmatteomanf!

**Comment by itsmatteomanf** [comment]
No problem! My pleasure.

This package solved a lot of issues I had with OG images :) I'd like some more customisation options, might open an issue to discuss if you are interested in those.

**Comment by delucis** [comment]
> I'd like some more customisation options, might open an issue to discuss if you are interested in those.

Happy to discuss. It’s a bit tricky to balance flexibility with a simple API with this approach because we need to translate a user’s options to commands for CanvasKit, but happy to hear ideas.

**Comment by itsmatteomanf** [comment]
> It’s a bit tricky to balance flexibility with a simple API with this approach

Yeah, exactly my thoughts. I'd say small customisations. But gonna open the Issue so it's specific :)

**Pull request merged**